### ISSUE TITLE
Tests for this gem

### ISSUE THREAD

**Issue by iraszl** [bumi/lnrpc#19]
I'm thinking to develop a comprehensive list of tests. Let me know if you have any thoughts on this subject. What framework should I use if any? I was thinking, it could just be a ruby script (ex. `test.rb`) that calls every function one by one, and verifies if they respond as expected. One could provide their own macaroon to test with.

**Comment by bumi** [comment]
yeah! that's important. We could also use the [Alby simulation network](https://github.com/getAlby/lightning-browser-extension/wiki/Test-setup) to do full integration tests. 

As a framework I'd use [rspec](https://rspec.info/) or simply [minitest](https://github.com/minitest/minitest).
